Sources/Sentry/include/SentryFileManager.h

```diff
@@ -55,6 +56,11 @@ SENTRY_NO_INIT
 
 - (void)deleteOldEnvelopeItems;
 
+/**
+ * Only used for testing.
```

Comment:
Could we create a `SentryFileManager+Test.h` in the test target to expose this?

Reply:
The FileManager isn't public, so I often don't add the extra header file. It's just extra work. Still, if you'd like, we could add one.

Reply:
@armcknight, I'm going to merge this PR and maybe open another PR, depending on your reply.